hw14/test_human.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+
+def test_check_age(create_human):
+    human = create_human
+    human.grow()
+    expected_age = 26
+    assert human.age == expected_age, 'Age did not change'
+
+
+def test_check_difference_age_after_grow(create_human_with_custom_params):
+    human = create_human_with_custom_params('Olga', 19, 'female')
+    start_age = human.age
+    human.grow()
+    after_year_age = human.age
+    expected_difference = 1
+    assert expected_difference == after_year_age - start_age, 'Difference did not change'
+
+
+@pytest.mark.parametrize('age, status', [(98, 'alive'), (100, 'dead')])
+def test_check_status(create_human, age, status):
+    human = create_human
+    human._Human__age = age
+    human.grow()
+    expected_status = status
+    assert human._Human__status == expected_status, 'Status did not change'
```

Comment:
_Human__status   it is a private attribute which is designed not to use somewhere outside class